### ISSUE TITLE
Biomarker Entry Pages

### ISSUE THREAD

**Issue by DaniallMasood** [glygener/glygen-issues#1389]
https://hivelab.tst.biochemistry.gwu.edu/biomarker/api/biomarker/detail/AA7202-2

@sujeetvkulkarni The evidence section in this example and some other examples we have looked at are not being populated. The pubmed papers exist in citation and not showing up in the evidence section. We also have evidence that is not always a paper (eg ClinVar, GWAS). They are in the evidence_source of the api.

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood 
evidence_list is empty under evidence_source in json.

evidence_source : [
"evidence_list" : [
]
]

![Screenshot 2024-05-30 at 11 20 17 AM](https://github.com/glygener/glygen-issues/assets/13891482/e9bb48e6-8009-4a7b-a0ca-b9816b85f929)

Data is expected in below format,
![Screenshot 2024-05-30 at 11 24 34 AM](https://github.com/glygener/glygen-issues/assets/13891482/ce5eec80-04bb-4694-bb21-1d4dc404f2d7)

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni yes not all evidence sources have evidence text associated with them. is it possible to still list the evidence source with the evidence list being empty?

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood Evidence for what will be the question. In case it points to multiple publications, then all such publications should get listed under Publications which comes from citation:[] array in json.

Technically you can add empty strings in the evidence attribute to list evidence source. But not recommended.
```
"evidence_source": [
  {
    "id": "10914713",
    "database": "Pubmed",
    "url": "https://pubmed.ncbi.nlm.nih.gov/10914713",
    "evidence_list": [
      {
        "evidence": ""
      }
    ],
    "tags": [
      {
        "tag": "condition"
      },
      {
        "tag": "best_biomarker_role"
      }
    ]
  }
]
```

**Comment by DaniallMasood** [comment]
the evidence is for the biomarker. When pulled from a source, such as ClinVar, there is no text associated with it. It is not a publication but we do want it to be displayed in the evidence section
